src/module-server/build-status-tracker.ts

```diff
@@ -0,0 +1,37 @@
+// This makes it so that errors thrown by the module server get re-thrown inside of runJS/loadJS.
+// This is necessary because the network is between runJS getting called and the module server, so errors do not propagate
```

Comment:
There is either an extra word or a word missing but I can't figure it out. 😅

Reply:
I _think_ it has all the words I meant for it to have. Maybe it would be clearer like this?

```suggestion
// This is necessary because the network is between the runJS call and the module server, so errors do not propagate
```

Reply:
Ah, got it. I agree with the change! 🙂